src/pages/HistoricalBalance/index.js

```diff
@@ -1,44 +1,31 @@
 import React, { useMemo } from 'react'
-import cx from 'classnames'
 import URLExtension from './URLExtension'
+import Page from '../../ducks/Page'
 import HistoricalBalance from '../../ducks/HistoricalBalance'
 import { parseUrl } from '../../ducks/HistoricalBalance/url'
-import MobileHeader from '../../components/MobileHeader/MobileHeader'
 import HelpPopup from '../../components/HelpPopup/HelpPopup'
-import styles from './index.module.scss'
 
-const MOBILE_CLASSES = {
-  wrapper: styles.mobile__title
-}
+const Help = () => (
+  <HelpPopup>
+    Enter any ERC-20 wallet's address and choose up to 5 assets for a detailed
+    breakdown of the wallet's balance over time.
+  </HelpPopup>
+)
 
 export const Title = () => (
   <>
-    Historical balance
-    <HelpPopup triggerClassName={styles.help}>
-      Enter any ERC-20 wallet's address and choose up to 5 assets for a detailed
-      breakdown of the wallet's balance over time.
-    </HelpPopup>
+    Historical balance <Help />
```

Comment:
```suggestion
    Historical balance{' '} <Help />
```
or add margin-left in styles